### DISCUSSION TITLE
option to clear HTML markup when switching from Bold editor to Plain editor?

### DISCUSSION THREAD

**Discussion by johnmahoney** [standardnotes/forum#2599] [Suggestions]
I was just experimenting with the Bold editor, and found that any note I create with it has some form of HTML markup (paragraph tags, etc). If I then switch to the plain editor, all of this markup is visible. 

Is there a way to to convert back to truly plain text when switching editors? Or another preferred method to pull plain text from a note created with the Bold editor?

**Comment by bryvin** [comment]
There isn't a direct way to convert back to plain text only, but my suggestion would be to first copy the text while in the Bold Editor (ctrl/cmd+a, ctrl/cmd+c) then switch to the plain editor and replace all the HTML with what you just copied (ctrl/cmd+a, ctrl/cmd+v).

This will place the text in regular format as expected and leaves line breaks in tact.

Note: There is no way to switch back and forth however; once you switch back to plain text and overwrite the HTML, your formatting will be lost and you cannot go back to the HTML version unless you switch to Bold editor and re-format manually.

**Comment by johnmahoney** [comment]
@bryvin That seems like a sensible workaround, thanks. You could also use Session History to revert back to the formatted version if you needed to, I realize (losing any subsequent changes you made after "converting" to plain text of course).

**Comment by MorgothSauron** [comment]
@bryvin Copy/paste works partially if I have image attached from FileSafe. I get the text, but when I paste I don't have the link to the image.
